_layouts/home.html

```diff
@@ -53,41 +47,74 @@ <h2>Lectures</h2>
               <p>QuantEcon hosts lecture series on economics, finance, econometrics and data science. All lecture series
                 are based on open source languages and open computing environments.</p>
               <p><a href="/projects/#filter=lecture" class="btn-learn-more">Browse lectures</a></p>
-
-              </p>
             </div>
           </div>
-
         </div>
+
         <div class="col-md-6 mt-5 mt-md-0">
 
           <div class="row">
 
-
             <div class="col-md-4 offset-md-0 d-none d-md-block">
               <p><img src="/assets/img/code-thumb.png" class="img-thumbnail" alt="Screenshot showing python code"></p>
             </div>
 
-
             <div class="col-md-8 offset-md-0">
               <h2>Code Libraries</h2>
               <p>QuantEcon has developed and maintains high-performance, open source, code libraries in Python and
-                Julia.
-              </p>
+                Julia.</p>
               <br>
               <p><a href="/projects/#filter=code" class="btn-learn-more">View libraries</a></p>
-
             </div>
           </div>
-
         </div>
-
       </div>
+    </div>
 
+    <div class="container">
+      <br>
+      <!-- <div class="section-title">
+        <h1>Books</h1>
+      </div> -->
+      <br>
+      <!-- Books's image and descriptions -->
+      <div class="row justisfy-content-center gx-5" data-aos="zoom-in" data-aos-delay="300">
 
+        <div class="col-md-6">
 
-    </div>
+          <div class="row">
+
+            <div class="col-md-4 offset-md-0 d-none d-md-block">
+              <p><img src="/assets/img/dp-image.png" class="img-thumbnail" alt="Screenshot showing a lecture page"></p>
+            </div>
+
+            <div class="col-md-8 offset-md-0">
+              <h3>Dynamic Programming</h3>
+              <p>This book is about dynamic programming and its applications in economics, finance, and adjacent
+                fields like operations research. It brings together recent innovations in the theory of dynamic
+                programming.</p>
+              <p><a href="https://dp.quantecon.org/" class="btn-learn-more">View online version</a></p>
+            </div>
+          </div>
+        </div>
+        <div class="col-md-6 mt-5 mt-md-0">
+
+          <div class="row">
+
+            <div class="col-md-4 offset-md-0 d-none d-md-block">
+              <p><img src="/assets/img/networks-image.png" class="img-thumbnail" alt="Screenshot showing python code">
```

Comment:
Same as above. This `alt` should be changed to something like

```suggestion
              <p><img src="/assets/img/networks-image.png" class="img-thumbnail" alt="Icon of Economic Networks book">
```